CODEOWNERS

```diff
@@ -13,6 +13,7 @@
 /specification/billing/ @wilcobmsft @asarkar84
 /specification/cdn/ @jorinmejia @yunhemsft @jessicl-ms @sinadell @rrahulms @t-bzhan
 /specification/cognitiveservices/ @felixwa @yangyuan
+/specification/cognitiveservices/data-plane/Language/ @rokulka @ChongTang @annatisch @heaths
```

Comment:
Would you please add team `@Azure/api-stewardship-board` as code owner, as well?

I didn't find the linked issue of API stewardship review. Let me know if there is one (would like to see previous meeting minutes there).

Reply:
I'm not aware of the issue as I recently took over.

Reply:
We would like to have some context of the change to continue the review (either you would need to book a meeting, or let me or Ted know whether this PR is a continuation from a preview review meeting).

Ref https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship

Reply:
Let me clarify: Currently, we have two open PR: 
1 - clu 2021-11-01-preview (this one) which has already gone through the API board review process, but for some reason, the PR was never merged into the main. Unfortunately, the engineer who drove this effort is on parental leave. @heaths reached out to get this done and I was assigned to work with him. That's why I do not have the full knowledge of board meeting minutes and the issue.

2 - clu 2022-03-01-preview which is under review by the API board. We presented this version on Feb 8th and you can find the design document, minutes as well as the recorded meeting. During the meeting I provided an overview of our service as well as the new capabilities.

https://github.com/Azure/azure-rest-api-specs/pull/17504
https://github.com/Azure/azure-rest-api-specs/issues/17505

I'll be happy to schedule a meeting to answer any questions that you might have.

Reply:
Thanks Farhad.
I was trying to find the meeting minutes (which should be similar to the one for 2022-03-01-preview at https://github.com/Azure/azure-rest-api-specs/issues/17505), so that I know what to focus on the change requests there. If you can find it, let us know (not necessary an issue, just notes mentioning what was discussed in the review meeting).
If you cannot find it, we might had to review the whole swagger offline (which could take more time).

Reply:
It is deployed. The problem with Cognitive Services in this repo is, unfortunately, with the unified approach they share an RP but are actually separate SDKs. The current validation doesn't really support this and we may need to think having a single RP and what ramifications that will have if/when we do. But the idea was to keep a set of common definitions for parameters and models (common.json, which TA in 2021-11-01-preview doesn't use but will in newer versions) to help facilitate a common feel across Language pillar services.

So, in effect, there's really no breaking change here, if that's the concern.

Reply:
@weidongxu-microsoft I'm on the SDK team and have been working on the SDK along with others. The service itself did go through an initial review quite a while back and has been under some development through the fall.

Reply:
I don't have any objections to merging. We can address the `x-nullable` concern in a future PR.